src/css/main.less

```diff
@@ -967,8 +976,69 @@ fieldset[disabled] .btn-info.active {
         font-weight: bold;
         font-size: @font-size-h3;
         line-height: @base-spacing-unit*2;
+        text-transform: uppercase;
         margin: @base-spacing-unit*2 0;
       }
+
+      ul.filters {
```

Comment:
Yeah – I thought we'll be extracting taking these into their own "modules" at some point anyway. But sure, the nesting here is *nasty* (eheh). Thinking about it, I'd say we should extract `.checkbox` outside of `.filters` – we might re-use checkbox styles somewhere else.